docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -7,41 +7,65 @@
 - **Higher-level API**. Toolpad operates at a higher abstraction level than Material UI. It sets healthy UI constraints that prevent common pitfalls and anti-patterns.
 - **Drag-and-drop builder**. Toolpad comes with a drag-and-drop builder to simplify UI building. The builder saves the UI as a YAML file, so you can even modify and version the visual output on your file system.
 - **Less boilerplate** so you can focus on the essential parts of the app.
-- **Run alongside existing code**. It integrates well with your existing code. You can use your database models, client libraries, secrets, and bespoke components directly.
+- **Run alongside existing code**. Toolpad integrates well with your existing code. You can use your database models, client libraries, secrets, and bespoke components directly.
 - **Single source of truth**. Because Toolpad runs alongside your existing code, it lets you maintain a single source of truth for all business logic.
 - **Integrates well with your development lifecycle**. Your project lives on your file system. You can use your own IDE, version control system, deployment target, CLI tools.
 
-After reviewing multiple products and approaches, a few recurring areas of concern were noticed. Let's take a closer look at them:
-
 ## Trade-offs with pro-code tools
 
-### Compromised delivery speed
+The key problems most pro-code tools have relative to Toolpad:
 
-Every organization aims to achieve fast delivery, but in order to accommodate various use cases, developers are often pushed to choose flexible pro-code solutions. However, this flexibility often comes at the cost of speed. Pro-code provides lower-level abstractions for greater control, but as use cases scale, they quickly involve a growing maintenance burden that can't be avoided.
+### Slower delivery speed
 
-### A lot of boilerplate
+Every organization wants to achieve fast delivery but in order to accommodate various use cases, you are pushed to choose flexible pro-code solutions.
 
+This flexibility comes at the cost of more **boilerplate** code.
 To connect backend data to the browser and set up data-display components, a substantial amount of boilerplate HTML, CSS, and JavaScript glue code is often required.
+As the use cases repeat themselves, you create abstractions but their maintenance burden quickly grows.
+
+Wouldn't it be great if you could use a tool that abstracts the repetitive parts of creating internal tools?
+
+### No direct manipulation of objects
+
+By the definition of a pro-code tool, you write code in an IDE and the UI renders in the browser, a different window.
+
+You can't **directly manipulate** the parameters, queries, etc. in the browser. It makes it harder to make changes on the application.
+
+Wouldn't it be great if it was possible to directly manipulate the app created in the browser, manipulating the data the closest possible to the source of truth?
 
-### Lower-level abstraction
+### Require specialized knowledge
```

Comment:
Essentially shortening the loop between writing code and consulting documentation.